docs/architecture/README.md

```diff
@@ -3,111 +3,184 @@ order: 3
-- **Api**: All the [apis](../api-guide/) that allowed to trigger executions for any system, and to interact with Kestra.
-- **UI**: The [User Interface](../user-interface-guide) (UI) is also served by the same webserver.
 
-The Webserver interacts mostly with Elasticsearch to deliver a rich API/UI. It also interacts with Kafka in order to trigger new executions, follow executions in real-time, etc...
+The **Webserver** offers 2 main modules in the same component:
+- **API**: All the [apis](../api-guide/) that allow to trigger executions for any systems, and to interact with Kestra.
+- **UI**: The [User Interface](../user-interface-guide) (UI) is also served by the webserver.
+
+The Webserver interacts mostly with the Repository to deliver a rich API/UI. It also interacts with the Queue in order to trigger new executions, follow executions in real-time, etc...
 
 
 ::: tip
-Each Application/Server (other than the Scheduler) can continue to function, provided Kafka and its Storage is up and running. ElasticSearch is only used to help provide our rich Web UI, and even if ElasticSearch is down, workloads can continue to process on Kestra.
+Each server component (other than the Scheduler) can continue to work as long as the Queue is up and running. The Repository is only used to help provide our rich Web UI, and even if the Repository is down, workloads can continue to process on Kestra.
 :::
 
-
 ## Plugins
+
 Kestra's core is not by itself able to handle many task types. We have therefore included a [Plugins](../../plugins) system that allows to develop as many task types as you need.
 A wide range of plugins is already available, and many more will be delivered by the Kestra team!
 
-## Repositories
-Repositories are the internal way to fetch data. On production servers, these will fetch data from ElasticSearch. 
+Plugins are also used to provide different implementations for Kestra internal components like its Internal Storage.
+
+## Deployment architecture
+
+Kestra is a Java application that is provided as an executable. You have many deployments options:
+- [Docker](../administrator-guide/deployment/docker)
+- [Kubernetes](../administrator-guide/deployment/kubernetes)
+- [Manual deployment](../administrator-guide/deployment/manual)
+
+At its heart, Kestra has a plugging system allowing to choose the dependency type that fit your needs.
+
+You can find bellow three example deployment architectures.
+
+### Small-sized deployment
+
+![Kestra Standalone Architecture](./architecture-standalone.svg "Kestra Standalone Architecture")
+
+For small-sized deployment, you can use Kestra standalone server which is a all-in one server component that allows to run all Kestra server components in a single process. This deployment architecture has no scaling capability.
+
+It can be used with a database as the only dependency. This allows to run Kestra with a minimal stack to maintain. We have for now 3 databases available:
```

Comment:
```suggestion
It can be used with a database as the only dependency. This allows running Kestra with a minimal stack to maintain. We have for now three databases available:
```